testing/mldb_python_plugin_test.cc

```diff
@@ -247,45 +247,45 @@ print "a"
     // check init output
     getResult = proxy.get("/v1/plugins/pyplugin/routes/lastoutput");
     cerr << "getResult = " << getResult << endl;
-    BOOST_CHECK_EQUAL(getResult.code(), 200);
-    BOOST_CHECK_EQUAL(getResult.jsonBody()["logs"][0]["c"].asString(), "testing pluging for MLDB!!");
+    BOOST_REQUIRE_EQUAL(getResult.code(), 200);
+    BOOST_REQUIRE_EQUAL(getResult.jsonBody()["logs"][0]["c"].asString(), "testing pluging for MLDB!!");
 
     status = jsonDecode<PolyStatus>(proxy.get("/v1/plugins/pyplugin").jsonBody());
     
     cerr << "status = " << jsonEncode(status) << endl;
-    //BOOST_CHECK_EQUAL(jsonEncode(status)["status"]["message"].asString(), "A-OK");
+    //BOOST_REQUIRE_EQUAL(jsonEncode(status)["status"]["message"].asString(), "A-OK");
     
 
     getResult = proxy.get("/v1/plugins/pyplugin/routes/hello");
     cerr << "getResult = " << getResult << endl;
-    BOOST_CHECK_EQUAL(getResult.code(), 200);
-    BOOST_CHECK_EQUAL(getResult.jsonBody()["how"].asString(), "are you");
+    BOOST_REQUIRE_EQUAL(getResult.code(), 200);
+    BOOST_REQUIRE_EQUAL(getResult.jsonBody()["how"].asString(), "are you");
     
     // make sure we're getting a 200 return code for empty dict and list
     getResult = proxy.get("/v1/plugins/pyplugin/routes/emptyList");
     cerr << "getResult = " << getResult << endl;
-    BOOST_CHECK_EQUAL(getResult.code(), 200);
-    BOOST_CHECK(getResult.jsonBody().isArray());
+    BOOST_REQUIRE_EQUAL(getResult.code(), 200);
+    BOOST_REQUIRE(getResult.jsonBody().isArray());
     
     getResult = proxy.get("/v1/plugins/pyplugin/routes/emptyDict");
     cerr << "getResult = " << getResult << endl;
-    BOOST_CHECK_EQUAL(getResult.code(), 404);
-//     BOOST_CHECK(getResult.jsonBody().isObject());
+    BOOST_REQUIRE_EQUAL(getResult.code(), 200);
```

Comment:
Here. An empty dict previously meant a 404. Now it's a valid 200 with the empty dict.